src/routes/profile.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from "@testing-library/react";
+import { Profile } from "./profile";
+
+describe("Route: Profile", () => {
+  it("renders profile header", () => {
+    render(
+      <Profile
```

Comment:
I'm letting people know that we don't really need to write tests for things inside of `/routes`

The reasoning is that these components are not re-useable and so your test is really just verifying that the page is rendering what you told it to, which you could check visually instead of writing an automated test for it. The automated tests in `/components` are more necessary because those components need to be flexible and render correctly in different contexts, so the tests verify that fact.

You can delete this file